pkg/agent/route/interfaces.go

```diff
@@ -80,4 +80,12 @@ type Interface interface {
 
 	// DeleteLocalAntreaFlexibleIPAMPodRule is used to delete related IP set entries when an AntreaFlexibleIPAM Pod is deleted.
 	DeleteLocalAntreaFlexibleIPAMPodRule(podAddresses []net.IP) error
+
+	// AddNodeIPs adds Node IPs into the ipset when a new Node joins the cluster.
+	// The ipset is consumed with encap mode when multicast is enabled.
+	AddNodeIPs(nodeIP4 net.IP, nodeIP6 net.IP) error
+
+	// DeleteNodeIPs deletes NodeIPs from the ipset when a Node leaves the cluster.
+	// The ipset is consumed with encap mode when multicast is enabled.
+	DeleteNodeIPs(nodeIP4 net.IP, nodeIP6 net.IP) error
```

Comment:
`AddRoutes` already passes nodeIPs, should we just update the ipset in that function? we also maintained an ipset for PodCIDR in that function. And we could add parameters to `DeleteRoutes` for deletion case.